### PR TITLE
test Jest

### PR DESCRIPTION
**Pull request by Ntarecedrick**

### CODE REVIEW

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Eli250) [inline coverage/clover.xml]: Do not push coverage folder

**Review by Eli250 — CHANGES_REQUESTED** [review]
Do not push anything in the coverage folder. Add that folder to gitignore.

File: coverage/clover.xml (hunk 1 above)
Comment (Ntarecedrick) [reply coverage/clover.xml]: okay let me add them to the gitignore

File: src/index.js [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+import express from 'express'
+import router from './routes/router';
+import authRoutes from './routes/auth';
+const app = express();
+app.use(express.json({limit: '500mb'}))
+// app.use(express.urlencoded({limit:'500mb'}))
 
+try {
+    mongoose.set('strictQuery', false)
+    mongoose.connect("mongodb://localhost:27017/serverDB", { useNewUrlParser: true });
```
Comment (Eli250) [inline src/index.js]: Don't expose your db URL. Use env variables.

**Review by Eli250 — CHANGES_REQUESTED** [review]
Don't expose your database URL. Use envs

File: src/index.js (hunk 2 above)
Comment (Ntarecedrick) [reply src/index.js]: i've completed that i've hidden it in the env variable

Comment (Eli250) [inline src/index.js]: You did not change this

**Review by Eli250 — CHANGES_REQUESTED** [review]

**Review by Eli250 — APPROVED** [review]

**Pull request merged**